packages/react-button/src/components/ToggleButton/useToggleButtonStyles.ts

```diff
@@ -15,7 +15,7 @@ export const toggleButtonClassNames: SlotClassNames<ButtonSlots> = {
  */
 export const toggleButtonClassName = toggleButtonClassNames.root;
 
-const useCheckedStyles = makeStyles({
+export const useCheckedStyles = makeStyles({
```

Comment:
These style hooks are only exported from react-button and not react-components. However, in your story you use these hooks to style the new compound toggle button

Reply:
Yes, that's on purpose right now, I want to start a conversation in a tech sync about exporting these individual hooks that I'd want to keep separate from this PR.